docusaurus.config.js

```diff
@@ -7,6 +7,9 @@ module.exports = {
   url: "https://docs.camunda.io",
   // baseUrl: "/camunda-cloud-documentation/",
   baseUrl: "/",
+  customFields: {
+    canonicalUrlRoot: "https://docs.camunda.io",
```

Comment:
Thinking ahead to unsupported sites -- they will not want to use their own host (unsupported.docs....) as the canonical. We'll want to point them over to the main site.